src/App/App.js

```diff
@@ -33,10 +48,13 @@ function App() {
       </header>
       <Switch>
         <Route exact path="/add/:side/:type">
-          <AddItem />
+          <AddItem onAddSubmit={addItemHandler} />
         </Route>
         <Route exact path="/edit/:itemId">
-          <EditItem balance={balanceLocalStorage} />
+          <EditItem
```

Comment:
I think it would suffice to only give the item to \<EditItem\> instead of the entire balance object